orttraining/orttraining/test/training_api/core/training_api_tests.cc

```diff
@@ -150,7 +150,7 @@ void TestLRSchduler(const std::string& test_file_name, float initial_lr, int64_t
 
   /// Load test data for learning rate schedulers.
   auto data_dir = MODEL_FOLDER "test_data_generation/lr_scheduler/";
```

Comment:
```
  constexpr const ORTCHAR_T* data_dir = MODEL_FOLDER ORT_TSTR("test_data_generation/lr_scheduler/");
```
Then in the next line you don't need to convert this string from char* to wchar_t*. However, when you define the MODEL_FOLDER macro, you also need to do it in such a way.

Reply:
Changed this a little... please review again